docs/content/en/4.providers/unsplash.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Unsplash Provider
+description: 'Nuxt Image has first class integration with Unsplash'
+navigation:
+  title: Unsplash
+---
+
+Integration between [Unsplash](https://unsplash.com/documentation#dynamically-resizable-images) and the image module.
```

Comment:
Maybe it is worth mentioning about [License](https://unsplash.com/license)? (mostly in good faith of using the service)

Reply:
```suggestion

Integration between [Unsplash](https://unsplash.com/documentation#dynamically-resizable-images) and the image module. See [Unsplash License](Unsplash photos are made to be used freely.) for what usage is permitted.
```